packages/desktop-gui/src/settings/experiments.jsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react'
+import { observer } from 'mobx-react'
+import ipc from '../lib/ipc'
+import { get } from 'lodash'
+import { parseExperiments } from '@packages/runner/src/lib/experiments'
+
+const openHelp = (e) => {
+  e.preventDefault()
+  ipc.externalOpen('https://on.cypress.io/experiments')
+}
+
+const openIssue = (number) => (e) => {
+  e.preventDefault()
+  ipc.externalOpen(`https://github.com/cypress-io/cypress/issues/${number}`)
+}
+
+const Experiments = observer(({ project }) => {
+  const resolvedEnv = get(project, 'resolvedConfig.env.EXPERIMENTS.value', '')
+  const experiments = parseExperiments(resolvedEnv)
+
+  return (
+    <div>
+      <a href='#' className='learn-more' onClick={openHelp}>
+        <i className='fas fa-info-circle'></i> Learn more
+      </a>
+
+      <div>
+        <p>
+        You can enable some beta features still under development by setting a special environment variable while running
+        Cypress. For example to enable experiments "featureA" and "featureB", open or run Cypress with:
+        </p>
+
+        <pre className='line-nums'>
+          <span>CYPRESS_EXPERIMENTS=featureA,featureB cypress open</span>
+          <span># or</span>
+          <span>CYPRESS_EXPERIMENTS=featureA,featureB cypress run</span>
+        </pre>
+
+        {resolvedEnv &&
+          <>
+            <p>The environment variable currently has the value</p>
+              <pre className='line-nums'>
+                <span>CYPRESS_EXPERIMENTS={resolvedEnv}</span>
+              </pre>
+          </>
+        }
+      </div>
+
+      <div>
+        <h3>{experiments.componentTesting ? '✔️' : '🚫'} component testing</h3>
```

Comment:
oh yeah, this is what i was talking about. i feel like the emoji are unclear, can we use "status: enabled/disabled" or something instead?

Reply:
look at the screenshots above, they are beautiful https://github.com/cypress-io/cypress/pull/6265#issuecomment-579402112

Reply:
but I am not against something like `status: enabled` or `status: disabled` labels instead.

Reply:
yeah, I think that would be clearer